### PR TITLE
qpdf: Update to 8.1.0

### PR DESCRIPTION
**Pull request by ryandesign**
#### Description

qpdf:
* Update to 8.1.0
* Remove obsolete patch
* Add new patch to prevent linkage with previously installed library

###### Type(s)

- [X] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
macOS 10.13.6 17G65
Xcode 9.4.1 9F2000 

###### Verification 
Have you

- [X] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [X] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [X] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [X] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [X] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [X] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

File: textproc/qpdf/Portfile [hunk 1]
```diff
@@ -14,13 +18,10 @@ long_description    QPDF is a command-line program that does structural, \
                     content-preserving transformations on PDF files.
 
 homepage            http://qpdf.sourceforge.net/
+#github.tarball_from releases
```
Comment (pmetzger) [inline textproc/qpdf/Portfile]: Why the comment?

**Comment by pmetzger** [comment]
LGTM, though I flagged what might have been a comment from while you were testing the change.

File: textproc/qpdf/Portfile (hunk 1 above)
Comment (ryandesign) [reply textproc/qpdf/Portfile]: To remind me to uncomment it and remove the sourceforge master_sites when the port is updated next time.

**Comment by pmetzger** [comment]
Merged. Thanks, @ryandesign!

**Pull request merged**